src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import styles from './Timer.module.css';
```

Comment:
You can remove this if there's no css file

Reply:
there is one now